### PR TITLE
Update the README and restructure the repo

### PR DESCRIPTION
**Pull request by asrivas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,25 @@
...
 
-You must be familar GCP and have your own project created.
+You must be familar GCP and have your own project with billing enabled.
 
-1. Set up your GCP project for Document AI following the [Setup Guide](https://cloud.google.com/document-ai/docs/setup)
-1. Enable the 'Document AI API' in your project in the [Cloud Console](https://console.cloud.google.com/ai/document-ai)
-1. Create a Python 3 AI Platform Notebook in your GCP project and clone this repository.
+1. Set up your GCP project for Document AI following the [Setup Guide][set_up].
+1. Enable the 'Document AI API' in your project in the Document AI [Platform][platform].
+1. Create a Python 3 notebook in your GCP project and clone this repository.
```
Comment (camushoilingma) [inline README.md:14]: Can we say something like "[Create](https://cloud.google.com/ai-platform/notebooks/docs/create-new) or use an existing instance of AI Platform Notebook with Python 3 using all default configurations." We can also say that Colab or any other Jupyter notebook would work but additional authentication setup may be required.

File: README.md [hunk 2]
```diff
@@ -1,33 +1,25 @@
...
+to be used with the Cloud [Document AI Platform][docai].
 
-![boxes](images/SampleOutput.png)
-![table](images/SampleFormFields.png)
+![boxes](resources/screenshots/SampleFormFields.png)
 
 ## Prerequisites 
 
-You must be familar GCP and have your own project created.
+You must be familar GCP and have your own project with billing enabled.
```
Comment (camushoilingma) [inline README.md:10]: Maybe modify it to:

Working knowledge of the following Google Cloud Platform products:
- [Google Cloud Storage](https://cloud.google.com/storage)
- [AI Platform Notebooks](https://cloud.google.com/ai-platform-notebooks) (Optional but would simplify the initial setup)
- Basic Understanding of [Google Document AI Concepts](https://cloud.google.com/document-ai/docs/basics) and [Processors](https://cloud.google.com/document-ai/docs/processor-overview)

File: README.md [hunk 3]
```diff
@@ -1,33 +1,25 @@
...
-2. Copy your processor id.
-
-![processorId](images/FormParserID.png)
-
-3. Find the cell where the processor variables are set, you will also need your GCP Project ID.
+1. Identify which form type or utility you would like to run through a processor.
+2. Create your processor using the [instructions][create_processor].
+3. Copy your processor id.
+![processorId](resources/screenshots/FormParserID.png)
+4. Find the cell where the processor variables are created, you will also need your GCP Project ID.
```
Comment (camushoilingma) [inline README.md:22]: NIT: Update the PROCESSOR_ID, PROJECT_ID and REGION variables in the notebook.

**Review by camushoilingma — COMMENTED** [review]
Just some minor recos on wording but looks good!

**Comment by asrivas** [comment]
> Just some minor recos on wording but looks good!

All great recommendations -- thank you!

**Review by camushoilingma — APPROVED** [review]

**Pull request merged**